src/splatting/Splatting.js

```diff
@@ -254,33 +267,65 @@ function createWorker(self) {
         var texdata_c = new Uint8Array(texdata.buffer);
         var texdata_f = new Float32Array(texdata.buffer);
 
+        let center_map = new Map();
+        // console.log(vertexCount);
+
         // Here we convert from a .splat file buffer into a texture
         // With a little bit more foresight perhaps this texture file
         // should have been the native format as it'd be very easy to
         // load it into webgl.
         for (let i = 0; i < vertexCount; i++) {
             // x, y, z
-            texdata_f[8 * i + 0] = f_buffer[8 * i + 0];
-            texdata_f[8 * i + 1] = f_buffer[8 * i + 1];
-            texdata_f[8 * i + 2] = f_buffer[8 * i + 2];
+            texdata_f[8 * i + 0] = f_buffer[9 * i + 0];
+            texdata_f[8 * i + 1] = f_buffer[9 * i + 1];
+            texdata_f[8 * i + 2] = f_buffer[9 * i + 2];
 
             // r, g, b, a
-            texdata_c[4 * (8 * i + 7) + 0] = u_buffer[32 * i + 24 + 0];
-            texdata_c[4 * (8 * i + 7) + 1] = u_buffer[32 * i + 24 + 1];
-            texdata_c[4 * (8 * i + 7) + 2] = u_buffer[32 * i + 24 + 2];
-            texdata_c[4 * (8 * i + 7) + 3] = u_buffer[32 * i + 24 + 3];
+            texdata_c[4 * (8 * i + 7) + 0] = u_buffer[36 * i + 24 + 0];
+            texdata_c[4 * (8 * i + 7) + 1] = u_buffer[36 * i + 24 + 1];
+            texdata_c[4 * (8 * i + 7) + 2] = u_buffer[36 * i + 24 + 2];
+            texdata_c[4 * (8 * i + 7) + 3] = u_buffer[36 * i + 24 + 3];
+
+            // cx, cy, cz, label (cluster center (x, y, z) and splat label/category)
+            // todo Steve: cx, cy, cz is read as Uint8 , therefore not accurate, need to find a way to read / parse & store them as actual Float32 values, just like x, y, z.
```

Comment:
Where are they written as uint8 instead of f32? I think the comment would be clearer if it pointed to the culprit

Reply:
Running out of space to store them😆 I only have 4 x uint8's left and I thought recovering a f32 from uint8 was possible somehow. But later I realized we actually don't have to store cx/cy/cz in each splats at all. I'll get rid of them and save some space for smth else we might store later.